examples/cisco/demo_bgp.pp

```diff
@@ -73,7 +73,11 @@
     enforce_first_as                       => false,
     event_history_cli                      => 'size_small',
     event_history_detail                   => 'size_medium',
+    # event_history_errors is not supported on all platforms
+    # event_history_errors                   => 'size_small',
```

Comment:
is there a version check condition example for demo manifests? I could not find it, can you please point me to one?